src/otx/core/data/dataset/visual_prompting.py

```diff
@@ -76,24 +86,21 @@ def _get_item_impl(self, index: int) -> VisualPromptingDataEntity | None:
 
         for annotation in item.annotations:
             if isinstance(annotation, dmPolygon):
-                mask = tv_tensors.Mask(polygon_to_bitmap([annotation], *img_shape)[0])
+                mask = tvMask(polygon_to_bitmap([annotation], *img_shape)[0])
                 mask_points = torch.nonzero(mask)
                 if len(mask_points[0]) == 0:
                     # skip very small region
                     continue
 
                 if torch.rand(1) < self.prob:
                     # get bbox
-                    bbox = tv_tensors.BoundingBoxes(
+                    bbox = tvBoundingBoxes(
                         annotation.get_bbox(),
-                        format=tv_tensors.BoundingBoxFormat.XYWH,
+                        format="xywh",
```

Comment:
`BoundingBoxFormat.XYWH` seems to be more reliable than a literal

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3769/commits/a425190b5050f81a5f10cfd3fd6432bfb3e43338